battleship.py

```diff
@@ -62,7 +62,14 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    a=[]

```

Comment:
Maintain a proper naming conventions for a variables.

Reply:
Done with the changes